snapshot-tests/turns/turn_5/orders.3

```diff
@@ -9,6 +9,7 @@ unit 152
 ;Workers (152), revealing faction, won't cross water, 10 nomads [NOMA],
 ;  186 silver [SILV]. Weight: 100. Capacity: 0/0/150/0. Skills: none.
 @work
+option template map
```

Comment:
Just to make sure we test the map format output, I enabled it for this turn, and then disabled it in the next turn.
I regenerated the templates (which is where these changes show up) for this turn with code from the current master.

Then coded my changes and verified that the output didn't change.